src/Plugin/Block/OpenseadragonBlock.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace Drupal\openseadragon\Plugin\Block;
+
+use Drupal\Core\Block\BlockBase;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Provides a 'OpenseadragonBlock' block from a IIIF manifest.
+ *
+ * @Block(
+ *  id = "openseadragon_block",
+ *  admin_label = @Translation("Openseadragon block"),
+ * )
+ */
+class OpenseadragonBlock extends BlockBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function defaultConfiguration() {
+    return [] + parent::defaultConfiguration();
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function blockForm($form, FormStateInterface $form_state) {
+    $form['iiif_manifest_url'] = [
+      '#type' => 'textfield',
+      '#title' => $this->t('IIIF Manifest URL'),
+      '#description' => $this->t('Absolute URL of the IIIF manifest to render.  You may use tokens to provide a pattern (e.g. "http://localhost/node/[node:nid]/manifest"'),
```

Comment:
You are still missing the closing (in the text) bracket here.
It displays like
> ... You may use tokens to provide a pattern (e.g. "http://localhost/node/[node:nid]/manifest"

Reply:
I have some code to list tokens here like it does for the title field if you so desire it. Or I can add that as a separate PR.

Reply:
if it's the same as scraping code from core to display the token list... i can hunt that down if you don't want it coming from you.

Reply:
I'll just leave this here 😉 
https://gist.github.com/whikloj/4fa01684dcbf50a1b75f38298ca80a68